src/python/grpcio/grpc/__init__.py

```diff
@@ -1868,6 +1868,23 @@ def alts_server_credentials():
     return ServerCredentials(_cygrpc.server_credentials_alts())
 
 
+def compute_engine_channel_credentials(call_credentials):
+    """Creates a compute engine channel credential.
+
+    This credential can only be used in a GCP environment as it relies on
+    a handshaker service. For more info about ALTS, see
+    https://cloud.google.com/security/encryption-in-transit/application-layer-transport-security
+
+    This channel credential is expected to be used as part of a composite
+    credential in conjunction with a call credential following the Application
```

Comment:
<i>Ideally</i>, the call credentials passed in should actually not use Application Default Credentials, and should instead use something that is guaranteed to authenticate as the VM's default service account.

Can we change this comment to, "in conjunction with a call credentials that authenticates the VM's default service account"

Reply:
Changed. Thanks for the clarification!